src/containers/offer-page/chat-dialog-window/ChatDialogWindow.tsx

```diff
@@ -68,6 +97,33 @@ const ChatDialogWindow: FC<ChatDialogWindow> = ({ chatInfo }) => {
     }
   }, [chatInfo.chatId, messagesLoad])
 
+  useEffect(() => {
+    if (createdChat._id) openChatInNewTab(createdChat._id as string)
+  }, [createdChat])
+
+  const closeChatWindow = () => setChatInfo(null)
+
+  const openChatInNewTab = (chatId: Pick<ChatResponse, '_id'> | string) => {
+    localStorage.setItem('currentChatId', chatId as string)
```

Comment:
do you need this as here and above?

Reply:
> do you need this as here and above?

Actually, yes, I haven't found any other way to transfer information between browser tabs other than localstorage. The information is deleted from localstorage, as soon as the user has been redirected to the desired chat

Reply:
> do you need this as here and above?

yup, without it, ts error appears